tests/detect-icmp-seq/test.yaml

```diff
@@ -0,0 +1,18 @@
+# *** Add configuration here ***
+
+args:
+- -k none
+
+checks:
+- filter:
+    count: 1
+    match:
+      event_type: alert
```

Comment:
If we have two rules, we should have two checks here, one for each rule sid. You can see how to do that check in the test bug-1401-01 (or many others)

Reply:
Now I think I get it! In the ``DetectIcmpSeqMatchTest01`` unit test it said that sid 2 should not alert so I thought its rule doesn't need a check, but instead I need to include the check with the count being 0, right?

Reply:
Yep, that's it :)